content/references/rippled-api/public-rippled-methods/account-methods/account_channels.md

```diff
@@ -149,32 +149,32 @@ An example of a successful response:
-| `cancel_after` | Unsigned Integer | _(May be omitted)_ Time, in [seconds since the Ripple Epoch][], of this channel's immutable expiration, if one was specified at channel creation. If this is before the close time of the most recent validated ledger, the channel is expired. |
-| `source_tag` | Unsigned Integer | _(May be omitted)_ A 32-bit unsigned integer to use as a [source tag](become-an-xrp-ledger-gateway.html#source-and-destination-tags) for payments through this payment channel, if one was specified at channel creation. This indicates the payment channel's originator or other purpose at the source account. Conventionally, if you bounce payments from this channel, you should specify this value in the `DestinationTag` of the return payment. |
-| `destination_tag` | Unsigned Integer | _(May be omitted)_ A 32-bit unsigned integer to use as a [destination tag](become-an-xrp-ledger-gateway.html#source-and-destination-tags) for payments through this channel, if one was specified at channel creation. This indicates the payment channel's beneficiary or other purpose at the destination account. |
+| Field                 | Type             | Description                       |
+|:----------------------|:-----------------|:----------------------------------|
+| `account`             | String           | The owner of the channel, as an [Address][]. |
+| `amount`              | String           | The total amount of [XRP, in drops][] allocated to this channel. |
+| `balance`             | String           | The total amount of [XRP, in drops][], paid out from this channel, as of the ledger version used. (You can calculate the amount of XRP left in the channel by subtracting `balance` from `amount`.) |
+| `channel_id`          | String           | A unique ID for this channel, as a 64-character hexadecimal string. This is also the [ID of the channel object](paychannel.html#paychannel-id-format) in the ledger's state data. |
+| `destination_account` | String           | the destination account of the channel, as an [Address][]. Only this account can receive the XRP in the channel while it is open. |
+| `settle_delay`        | Unsigned Integer | The number of seconds the payment channel must stay open after the owner of the channel requests to close it. |
+| `public_key`          | String           | _(May be omitted)_ The public key for the payment channel in the XRP Ledger's [base58][] format. Signed claims against this channel must be redeemed with the matching key pair. |
+| `public_key_hex`      | String           | _(May be omitted)_ The public key for the payment channel in hexadecimal format, if one was specified at channel creation. Signed claims against this channel must be redeemed with the matching key pair. |
```

Comment:
These have the same descriptions as before, but I've moved `settle_delay` up to be with the other guaranteed fields, and changed the whitespace so the columns match better in the MD source.

Reply:
"the destination account" - Capitalize

Reply:
Is there a difference between "May be omitted" and "Optional"?

Reply:
"Optional" is for requests. "May be omitted" is for responses.